plugin/parinfer.vim

```diff
@@ -151,23 +161,21 @@ com! -bar ToggleParinferMode cal parinfer#ToggleParinferMode()
 
 augroup parinfer
   autocmd!
-  autocmd InsertLeave *.clj,*.cljs,*.cljc,*.edn,*.hl,*.lisp,*.rkt,,*.ss call parinfer#process_form()
-  autocmd FileType clojure,racket,lisp,scheme nnoremap <buffer> <Tab> :call parinfer#do_indent()<cr>
-  autocmd FileType clojure,racket,lisp,scheme nnoremap <buffer> <Tab> :call parinfer#do_indent()<cr>
-  autocmd FileType clojure,racket,lisp,scheme nnoremap <buffer> <S-Tab> :call parinfer#do_undent()<cr>
-  autocmd FileType clojure,racket,lisp,scheme vnoremap <buffer> <Tab> :call parinfer#do_indent()<cr>
-  autocmd FileType clojure,racket,lisp,scheme vnoremap <buffer> <S-Tab> :call parinfer#do_undent()<cr>
+  execute "autocmd InsertLeave " . join(g:vim_parinfer_globs, ",") . " call parinfer#process_form()"
+  execute "autocmd FileType " . join(g:vim_parinfer_filetypes, ",") . " nnoremap <buffer> <Tab> :call parinfer#do_indent()<cr>"
+  execute "autocmd FileType " . join(g:vim_parinfer_filetypes, ",") . " nnoremap <buffer> <S-Tab> :call parinfer#do_undent()<cr>"
+  execute "autocmd FileType " . join(g:vim_parinfer_filetypes, ",") . " vnoremap <buffer> <Tab> :call parinfer#do_indent()<cr>"
+  execute "autocmd FileType " . join(g:vim_parinfer_filetypes, ",") . " vnoremap <buffer> <S-Tab> :call parinfer#do_undent()<cr>"
```

Comment:
There may well be better/cleaner ways to avoid the repeated joins.

Reply:
Turns out there is a good pattern, just a `let s:foo = expr` statement.  Still, leaving as-is for now.